src/ceph-volume/ceph_volume/api/lvm.py

```diff
@@ -906,6 +907,12 @@ def get(self, lv_name=None, vg_name=None, lv_path=None, lv_uuid=None, lv_tags=No
             raise MultipleLVsError(lv_name, lv_path)
         return lvs[0]
 
+    def dcopy(self):
+        copy = Volumes(populate=False)
+        for vol in self:
+            copy.append(vol)
+        return copy
+
```

Comment:
I assume this is no longer needed?

Reply:
It is not. But I think it's nice to have since there is no method to perform a deep copy and I would like to discourage creating new list of LVs from scratch by having this method since creating new list is more expensive.

Reply:
Ok; removed dcopy().